{
  "comments": [
    {
      "key": {
        "uuid": "9fdfeff1_c0dd87e0",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 11,
      "author": {
        "id": 12898
      },
      "writtenOn": "2019-01-31T03:32:19Z",
      "side": 1,
      "message": "At all?  Which version of Fedora are affected?",
      "range": {
        "startLine": 11,
        "startChar": 58,
        "endLine": 11,
        "endChar": 64
      },
      "revId": "471eac845afabc94ab4ededeb2004c828f781b61",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "9fdfeff1_00d06fc8",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 15,
      "author": {
        "id": 12898
      },
      "writtenOn": "2019-01-31T03:32:19Z",
      "side": 1,
      "message": "I don\u0027t see this, did you mean the setup.py bump from 1.3 to 1.8?",
      "range": {
        "startLine": 15,
        "startChar": 0,
        "endLine": 15,
        "endChar": 39
      },
      "revId": "471eac845afabc94ab4ededeb2004c828f781b61",
      "serverId": "4a232e18-c5a9-48ee-94c0-e04e7cca6543",
      "unresolved": false
    }
  ]
}